scripts/check_frontend_test_coverage.py

```diff
@@ -64,6 +64,8 @@
     'rte-helper-modal.controller.ts',
     'rule-type-selector.directive.ts',
     'translation-file-hash-loader-backend-api.service.ts',
+    # TODO(#19499): Add missing tests.
+    'translation-suggestion-review-modal.component.ts',
```

Comment:
@nikitaevg Why did this change? It's a lint PR, it should not be affecting tests...

Reply:
This one is funny

See [this](https://github.com/oppia/oppia/blob/345b7d350025a4e5172ea1b0a55790e89d8d0c3f/core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts#L586) line? If it's a single line, it's tested.

Once it's transformed into
```
      htmlString = Array.from(imgElements)
        .map(img => img.outerHTML)
        .join('');
```
the `map` part is not tested, because apparently imgElements is always empty in tests.

I'll add tests after this PR is merged.

Reply:
Ohh, interesting, haha. OK sg, thanks!